riotfile.py

```diff
@@ -273,6 +273,15 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             command="pytest {cmdargs} tests/tracer/ --ignore=tests/tracer/test_http.py",
             venvs=[
                 Venv(pys=select_pys()),
+                # This test variant ensures all tracer tests are compatible with 128bit trace ids.
+                # Once DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED is set to True by default this test variant
+                # should be removed (or DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED should be set to false).
+                Venv(
+                    pys=select_pys(),
+                    env={
+                        "DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED": "True",
```

Comment:
this means we're only running the http tests with `DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED=True` is that intentional?

should it be 
```suggestion
                        "DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED": ["", "True"],
```
?

Reply:
Yes by default this configuration is false. I avoided this change due to the ci costs.

 It is not explicit but all the other tracer tests run with `"DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED": "false"`

Reply:
Different code paths are taken so I think it's worth running both. CI cost can be addressed if it becomes an issue.